fdbserver/WorkerInterface.actor.h

```diff
@@ -147,6 +147,8 @@ struct ClusterControllerFullInterface {
 	RequestStream<struct GetServerDBInfoRequest>
 	    getServerDBInfo; // only used by testers; the cluster controller will send the serverDBInfo to workers
 
+	RequestStream<struct UpdatePeerHealthRequest> updatePeerHealth;
```

Comment:
needs to update initEndpoints() and hasMessage() function. 
Use `getServerDBInfo` as a reference.